index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
```

Comment:
Good job so far! 👍 

- [ ] - Please check the name of your file CSS in HTML. Change _styles.css_ to _style.css_ because on your webpage doesn´t show any style.